orchestrate.py

```diff
@@ -327,7 +332,9 @@ def main():
 
     handlers = [
         (r"/api/spawn/?", APISpawnHandler),
-        (r"/api/stats/?", APIStatsHandler)
+        (r"/api/stats/?", APIStatsHandler),
+        (r"/stats/?", RedirectHandler, {"url": "/api/stats"}),
+        (r"/info/?", InfoHandler),
```

Comment:
How so? `/info` only accesses `/api/stats`, which is already exposed in that case. This just makes the stats endpoints consistent.